pages/badge-single.jsx

```diff
@@ -0,0 +1,217 @@
+var React = require('react'),
+    HeroUnit = require('../components/hero-unit.jsx'),
+    SocialShare = require('../components/social-share.jsx'),
+    BadgeHorizontalIcon = require('../components/badge-horizontal-icon.jsx'),
+    RequirementsList = require('../components/requirement-list.jsx'),
+    Badge = require('../components/badge.jsx'),
+    Link = require('react-router').Link,
+    $ = require('jquery'),
```

Comment:
React and ES2015 remove the need for jQuery and underscore, so it's worth taking these back out.